src/track/track.h

```diff
@@ -282,7 +275,10 @@ class Track : public QObject {
             mixxx::CueInfoImporterPointer pCueInfoImporter);
     ImportStatus getCueImportStatus() const;
 
-    bool isDirty();
+    QMap<int, MacroPointer> getMacros() const;
```

Comment:
Build fails because `MacroPointer` is not declared in scope.

Reply:
huh, build doesn't fail for me?

Reply:
Check the CI.